src/components/map/UserCard.tsx

```diff
@@ -21,6 +21,8 @@ const UserCard = ({ user, onClose }: UserCardProps) => {
   const [lastActive, setLastActive] = useState(user.lastActive); // TODO: setLastActive when user is active?
   const [location, setLocation] = useState(user.lastActive.location);
   const [relativeTimeString, setRelativeTimeString] = useState('...');
```

Comment:
what is ...?

Reply:
probably a loading indicator?

Reply:
BTW there is a util function for getting the relative time string